cmd/index_add.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2022 The Falco Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/falcosecurity/falcoctl/pkg/index"
+	"github.com/falcosecurity/falcoctl/pkg/options"
+)
+
+type indexAddOptions struct {
+	*options.CommonOptions
+}
+
+func (o *indexAddOptions) Validate(args []string) error {
+	if _, err := os.Stat(falcoctlPath); os.IsNotExist(err) {
+		err = os.Mkdir(falcoctlPath, 0o700)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
```

Comment:
Note for the reviewer: not sure this should be the responsibility of this command. Couldn't find any other place to put it

Reply:
I agree that it should be a shared utility, not an _index_ function. It's ok for me just to put a comment (i.e. `fixme(loresuso): we should move this logic elsewhere`) then refactor it later.